### DISCUSSION TITLE
How to convert TMP files to plot files

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#1618] [Q&A]
Hello, today I had an issue,
I accidentally clicked on chia setup while chia app was running thus the app was terminated (maybe also fix that in future updates to check if app is open and ask user...) and so I lost the 2 tmp files that were almost ready (they were being copied to the hdd from the ssd. 
Now I have 2 TMP files 103GB each that are complete. Is there a way to recover these files and make them plot files. The procedure was actually terminated while app was copying these 2 files to the hdd.

Thanks

**Comment by hoffmang9** [comment]
If the files did complete you can often just rename them with a `.plot` extension and they will work.

**Reply by GeorgePapageorgakis** [reply]
Thank you that worked.
